### DISCUSSION TITLE
Is there a way to limit the API request rate?

### DISCUSSION THREAD

**Discussion by DerEwige** [Ride-The-Lightning/RTL#1066] [Q&A] [answered]
My RTL freezes (overloads) my eclair node when it tries to load the invoices.
(When I go to lightning -> transactions)

The node CPU goes up to 100% (4 cores) and I get time out errors in the logs for http://localhost:8080/listinvoices

I asume that RTL has problem handling over a million of invoices.
Is there an option in the config to tell RTL to load the data slower?

**Comment by saubyk** [comment, marked as answer]
This item will get addressed with the below issue:
https://github.com/Ride-The-Lightning/RTL/issues/1067
